sys/auto_init/netif/auto_init_ng_at86rf2xx.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/*
+ * @ingroup auto_init_ng_netif
+ * @{
+ *
+ * @file
+ * @brief   Auto initialization for nx_at86rf2xx network interfaces
+ *
+ * @author  Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#include "board.h"
+#include "net/ng_nomac.h"
+#include "net/ng_netbase.h"
+
+#include "ng_at86rf2xx.h"
+#include "ng_at86rf2xx_params.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#ifdef MODULE_NG_AT86RF2XX
+
+/**
+ * @brief   Define stack parameters for the MAC layer thread
+ * @{
+ */
+#define AT86RF2XX_MAC_STACKSIZE     (KERNEL_CONF_STACKSIZE_DEFAULT)
```

Comment:
Well, the MAC implementation calls a driver's functions for e.g., send, receive, etc. So this is low-level driver dependent and will be different for different devices. That's why it is defined here. If defined by nomac, it would have to be a size that fits all drivers, making possible optimization impossible.